### PR TITLE
[MagicBot] Bumping package version

### PR DESCRIPTION
**Pull request by DylanBaker**
#### This pull request was created automatically 🎉

    Before merging this PR:
    - [ ] Verify that all the tests pass.
    - [ ] Tag a release

### CODE REVIEW

File: integration_tests/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.0
+dbt-spark==0.20.0
+dbt-spark[PyHive]==0.20.0
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:3]:
```suggestion
dbt~=0.20.0
dbt-spark~=0.20.0
dbt-spark[PyHive]~=0.20.0
```

File: integration_tests/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt~=0.20.0
+dbt-spark~=0.20.0
+dbt-spark[PyHive]~=0.20.0
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:3]:
```suggestion
dbt==0.20.1
dbt-spark==0.20.1
dbt-spark[PyHive]==0.20.1
```

File: integration_tests/requirements.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.1
+dbt-spark==0.20.1
+dbt-spark[PyHive]==0.20.1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:3]:
```suggestion
dbt==0.20.0
dbt-spark==0.20.1
dbt-spark[PyHive]==0.20.1
```

File: integration_tests/requirements.txt [hunk 4]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.0
+dbt-spark==0.20.1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:2]:
```suggestion
```

File: integration_tests/requirements.txt [hunk 5]
```diff
@@ -1,3 +1,2 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.0
+dbt-spark[PyHive]==0.20.1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:2]:
```suggestion
dbt==0.20.1rc1
dbt-spark==0.20.1
dbt-spark[PyHive]==0.20.1
```

File: integration_tests/requirements.txt [hunk 6]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.1rc1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:1]:
```suggestion
dbt-core==0.20.1rc1
```

**Comment by epkworkrise** [comment]
Really need this pushed through, its affecting package dependencies for the ads_reporting dbt module.

File: integration_tests/requirements.txt (hunk 6 above)
Comment (fivetran-joemarkiewicz) [reply integration_tests/requirements.txt:1]:
```suggestion
dbt==0.20.1-rc1
```

File: integration_tests/requirements.txt [hunk 7]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.1-rc1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:1]:
```suggestion
dbt==0.20.1rc1
```

File: integration_tests/requirements.txt [hunk 8]
```diff
@@ -1,3 +1,3 @@
...
+dbt-spark==0.20.1
+dbt-spark[PyHive]==0.20.1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt]:
```suggestion
dbt==0.20.1rc1
dbt-spark==0.20.1rc1
dbt-spark[PyHive]==0.20.1rc1
```

File: integration_tests/requirements.txt [hunk 9]
```diff
@@ -1,3 +1,3 @@
-dbt==0.19.1
-dbt-spark==0.19.1
-dbt-spark[PyHive]==0.19.1
\ No newline at end of file
+dbt==0.20.1rc1
+dbt-spark==0.20.1
+dbt-spark[PyHive]==0.20.1
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:3]:
```suggestion
dbt-spark==0.20.1rc1
dbt-spark[PyHive]==0.20.1rc1
```

**Comment by fivetran-joemarkiewicz** [comment]
> Really need this pushed through, its affecting package dependencies for the ads_reporting dbt module.

Hi @epkworkrise this is a PR our team is eagerly awaiting to merge! We were holding off the merge because of an issue within dbt-spark 0.20.0. It looks like dbt-spark and dbt-core 0.20.1rc1 was just pre-released and it looks like our integration runs just passed. I will look closer into this to determine if this pre-release for spark compatibility will be sufficient. I hope to update back here soon on further updates!

**Pull request merged**